### ISSUE TITLE
Enhance a11y for `Menu`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1250]
path: stories/components/overlay/menu.stories.tsx

### Description
2 accessibility violations were found in the `Menu`.

#### [withCommand](https://yamada-ui.github.io/yamada-ui/?path=/story/components-overlay-menu--with-command)

Expected the HTML found at $('.ui-button') to have no violations:
    <button class="ui-button ui-icon-button ui-menu css-b77p09" type="button">

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

#### [withIcon](https://yamada-ui.github.io/yamada-ui/?path=/story/components-overlay-menu--with-icon)

Expected the HTML found at $('.ui-button') to have no violations:
    <button class="ui-button ui-icon-button ui-menu css-b77p09" type="button">

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

**Comment by yamao-latte** [comment]
@hirotomoyamada

Hi! Can I work on this issue?

**Comment by hirotomoyamada** [comment]
@yamao-latte 

Hi, thanks for the comment.

Yes, please!
